spec/features/posts_index_by_user_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'User Post Index', type: :feature do
```

Comment:
- [OPTIONAL] This test is also missing in your post index page. 
`Tests that redirects to the post's show page when you click on a post`. I made it optional because, it escaped the sight of the previous reviewer. Kindly treat it as being important.